lecsum-fe/pages/retry_exam.py

```
@@ -0,0 +1,474 @@
+    st.session_state['retry_quizzes'] = retry_data['quizzes']
+    st.session_state['retry_answers'] = {}
+    st.session_state['retry_result'] = None
+    st.session_state['retry_step'] = 'taking'
+    # 처리 후 삭제
+    del st.session_state['current_retry_quiz']
+
+
+def create_retry_quiz(quiz_ids):
+    """재시험 생성 API 호출"""
+    try:
+        response = requests.post(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)